tests/integration/sriov_test.py

```diff
@@ -109,6 +109,17 @@ def test_sriov_remove_vf_config(sriov_iface_vf):
     assertlib.assert_state(sriov_interface)
 
 
+@pytest.mark.xfail(
```

Comment:
The xfail has a `condition`, I am wondering whether that could be used to check whether `eth1` support SRIOV or not via sysfs. Not related to this PR.

Reply:
If I am not wrong, there was a command to check if an interface supports SR-IOV. Maybe we could use it and check the result. Usinf skipif instead of xfail will improve the CI time a bit when SR-IOV is not supported.